src/apis/crud.py

```diff
@@ -0,0 +1,52 @@
+from src.apis.models import Product
+from src.app import db
+
+
+def add_product(parsed_product):
+    """ 
+    Adds a products into the database
+    It also checks if already exists by the URL (not the best way for sure)
+    """
+    url = parsed_product["url"]
+    img_url = parsed_product["img_url"]
+    brand = parsed_product["brand"]
+    title = parsed_product["title"]
+    price = parsed_product["price"]
+    product_exists = Product.query.filter_by(url=url).all()
+    if product_exists:
+        product = []
```

Comment:
Algum motivo para não retornar o produto existente aqui? Porque no `products.py` é feita uma comparação com uma lista vazia, e aí fazendo isso aqui você poderia evitar esse tratamento.

Reply:
Pensei um pouco sobre essa questão, mas se eu retornar o product nesse caso, no products.py como irei verificar se o produto já existe? 
Que se o produto for retornado quando ele já existir, não teria muita diferença entre o retorno de quando foi criado, e o retorno de quando ele já existir. Eu poderia mudar a lógica para retornar o status_code junto com o produto, mas já teria um pouco da lógica da API dentro do crud.py, que não é bem a ideia.
Mas fico esperando para conversarmos melhor sobre essa questão :D